plugins/services.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: services
+spec:
+  version: v0.1.2
+  homepage: https://github.com/feloy/kubectl-services
+  shortDescription: Explore ingresses and services and their associated backends
```

Just a suggestion:
```suggestion
  shortDescription: Status for ingresses, services, and their backends
```
If at all  possible, stay below 50 chars, because anything beyond is usually truncated.